introduction.md

```diff
@@ -0,0 +1 @@
+isgarcia, esto es una prueba tras hacer un fork de un proyecto github y clonarlo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.